config/locales/en.yml

```diff
@@ -228,6 +223,25 @@ en:
       heading_1: Apply for Legal Aid
       welcome_home: Welcome Home
   providers:
+    start:
```

Comment:
This section clashed with a section I'd added in a rebase - I thought I'd fix that by taking the opportunity to moving it to the start of the `providers` section as it's the start or that journey.